Final_project_2/frontend/css/style.css

```diff
@@ -485,26 +480,34 @@ address{
   margin-left: 80px;
   margin-bottom: 70px;
 }
+
 address .fa {
   display: block;
   font-size: 33px;
   margin-bottom: 20px;
 }
 
+.address-item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
 
-.align {
+.half-width {
   width: 50%;
   float: left;
-  margin-bottom: 80px;
-  
 }
 
-.form-control {
-  display: block;
-  width: 80%;
+.contact-form-column {
```

Comment:
Renamed for the same reason - `form-control` is a very generic name that could easily be used somewhere else, and then the other developer is like "why does my new form have these weird properties??"